CMakeLists.txt

```diff
@@ -97,8 +97,10 @@ set(LIBMEM_DEPS
 
 if (LIBMEM_BUILD_STATIC)
 	add_library(libmem STATIC ${LIBMEM_SRC})
+	target_compile_definitions(libmem PUBLIC LM_STATIC)
```

Comment:
```suggestion
	target_compile_definitions(libmem PUBLIC LM_EXPORT LM_STATIC)
```

Reply:
I've just added LM_EXPORT to both static and shared library. Seems like a good change that keeps backwards compatibility.